internal/internal_activity.go

```diff
@@ -105,12 +105,15 @@ type (
 		startedTimestamp   time.Time
 		taskList           string
 	}
+
+	// context.WithValue need this type instead of basic type string to avoid lint error
+	contextKey string
 )
 
 const (
-	activityEnvContextKey          = "activityEnv"
-	activityOptionsContextKey      = "activityOptions"
-	localActivityOptionsContextKey = "localActivityOptions"
+	activityEnvContextKey          contextKey = "activityEnv"
```

Comment:
Does this syntax makes activity and localActivity keys of contextKey type as well?

Reply:
nop, only activityEnvContextKey is of contextKey

Reply:
I changed their order to make it looks even safer